code/modules/scripting/Implementations/Telecomms.dm

```diff
@@ -244,7 +244,7 @@ var/list/allowed_custom_spans = list(SPAN_ROBOT,SPAN_YELL,SPAN_ITALICS,SPAN_SANS
 		signal.data["verb_yell"]	= interpreter.GetCleanVar("$yell")
 		signal.data["verb_exclaim"]	= interpreter.GetCleanVar("$exclaim")
 		var/list/setspans 			= interpreter.GetCleanVar("$filters") //Save the span vector/list to a holder list
-		setspans &= allowed_custom_spans //Prune out any illegal ones. Go ahead, comment this line out. See the horror you can unleash!
+//		setspans &= allowed_custom_spans //Prune out any illegal ones. Go ahead, comment this line out. See the horror you can unleash!  //Release the SPANS!
```

Comment:
This is such a bad idea that I cannot even begin to describe how bad of an idea this is. You are a monster if you merge this. But, I do not play here, so it is your call :P